Gems/EMotionFX/Code/EMotionFX/Source/MotionEvent.cpp

```diff
@@ -62,6 +79,24 @@ namespace EMotionFX
     {
     }
 
+    MotionEvent& MotionEvent::operator=(const MotionEvent& other)
+    {
+        Event::operator=(other);
+        m_startTime = other.m_startTime;
+        m_endTime = other.m_endTime;
+        m_isSyncEvent = other.m_isSyncEvent;
+        return *this;
+    }
+
+    MotionEvent& MotionEvent::operator=(MotionEvent&& other)
+    {
+        Event::operator=(other);
```

Comment:
update: Need to `AZStd::move` `other` here too otherwise it will call the normal assignment operator - as this stuff is complex we really should add tests to cover it